index.html

```diff
@@ -279,6 +279,27 @@ <h3 class="panel-title">Shawn Lim</h3>
             </div>
           </div>
         </div>
+        <!-- Jerry -->
+        <div class="col-md-4">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <!--  Add your name here -->
+              <h3 class="panel-title">Jerry Tan</h3>
+            </div>
+            <div class="panel-body">
+              <!-- Add your image path in the src attribute -->
+              <img src="img/jerry.jpg" alt="Jerry" class="profile-photo img-rounded">
```

Comment:
@jerrygraced your image is not showing up. Probably because your file is called `jerry.JPG` while your `src` points to `jerry.jpg`. Try changing that and preview the result [here](https://htmlpreview.github.io/?https://raw.githubusercontent.com/jerrygraced/ga-wdi-class/gh-pages/index.html).